src/Message/UserMessage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Message;
+
+use PhpLlm\LlmChain\Message\Content\ContentInterface;
+use PhpLlm\LlmChain\Message\Content\Image;
+use PhpLlm\LlmChain\Message\Content\Text;
+
+final readonly class UserMessage implements MessageInterface
+{
+    /**
+     * @var list<ContentInterface>
+     */
+    public array $content;
+
+    public function __construct(
+        ContentInterface ...$images,
```

Comment:
Why is this variable named `$images`?

Reply:
Because third time rebuilding. Thanks for pointing this out .